docs/test-plans/001-Big-Blocks/test-cases/tc-005-das-past.md

```diff
@@ -0,0 +1,55 @@
+# Test-Case #005 - DA nodes are DASing past headers faster then validators produce new ones
+
+## Pre-Requisites:
+
+1. Every validator has enough funds in the account
+2. The chain has created the first block
+3. Every validator has enough peers
+4. Validators’ set is not changing during test execution
+   1. All validators are created during genesis
+5. DA Nodes amount is not changing during the test execution
+   1. We gracefully add DA Nodes as the first block is produced
+
+## Steps for each of the validators:
+
+1. Setups network with:
+   1. `I` mb of bandwidth
+   2. `J` milliseconds of network latency
+2. Generates and broadcasts
+   1. `X` kb of random data
+   2. `Y` times
+   3. IP and Genesis Hash for DA Bridge nodes
+3. Checks the block size is bigger then 3.5 MiB
+
+## Steps for each of the DA Bridge nodes:
+
+1. Setups network with:
+   1. `I` mb of bandwidth
+   2. `J` milliseconds of network latency
+2. Connects to respective Validator
+3. Shares the genesis hash and ip to Full / Light Nodes
+4. Check that it is synced
+5. Broadcasts new blocks to the DA network
+
+## Steps for each of DA Full / Light nodes:
+
+1. Setups network with:
+   1. `I` mb of bandwidth
+   2. `J` milliseconds of network latency
+2. Receives the trusted genesis hash and ip from Bridge Nodes
+3. Waits until `N` amount of block has been produced by the chain
+4. Starts DASing the chain afterwards
+5. Checks that it can:
+   1. DASes the past headers faster then new blocks are produced (\*)
+   2. DASes the current header in parallel
```

Comment:
I don't think this is necessary in the scope of this test. Basically we **only** need to test how quickly new node can go 0 -> network head when sampling (without errors or gaps) at least for this test case.

Reply:
Ok. I'll remove